src/commands/info/about.ts

```diff
@@ -22,7 +22,7 @@ const execute = async (interaction: ChatInputCommandInteraction) => {
   }
 
   const aboutInfoEmbed = defaultEmbed(DefaultEmbedType.NORMAL)
-    .setTitle('Reimagine your Community')
+    .setTitle(`${t('title.title', { lng: 'en', ns: 'about_command' })}}`)
```

Comment:
why is the title in the title category?

just key `title`, no category needed.